wasp-ai/src/client/project/converters.js

```diff
@@ -19,7 +21,8 @@ export function getStatusName(status) {
   }
 }
 
-export function getStatusText(status) {
+// converts project's status to displayable text
+export function convertProjectStatusToText(status) {
```

Comment:
Ok, this one is not bad. Maybe I would go with `projectStatusToDisplayableText` or `projectStatusToPrettyText` and then you can ditch the comment.

Reply:
sounds good, but part of me now wants to use the same `getXforY` naming structure as the other util functions to be consistent. something like e.g. `getDisplayableTextForProjectStatus`

Reply:
Yup that also sounds good, why not.

There is a slight difference I think: Here it feels like conversion is 1 to 1 -> you are not loosing any info really, you are converting from ugly to nice string, from id to string.

On the other functions, you are getting a style (class name) for a concept. Project color -> style. Ok, that sounds 1 on 1. But what about project status to style? There is information loss there. Maybe different statuses have same styles. It would certainly be weird to go back from style to status. Maybe also from style to color name, although less weird, depends on the situation.

So getXforY sounds good to me when there is information loss in this "conversion", because then it is not really a conversion, it is instead "describe me this thing (Y) from this perspective (X)" aka "give me X for Y".

Btw this is all very subtle and whatever you do would be ok really, be it `getXForY` or `XtoY`, but this is kind of reasoning I used while deciding what to use.